tests/api/test_reference.py

```diff
@@ -317,7 +321,7 @@ def test_reference_large(self, db, auth_headers, populate_test_mod_reference_typ
             assert response['mesh_terms'][0]['heading_term'] == "hterm"
 
             # cross references in the db?
-            xref = db.query(CrossReferenceModel).filter(CrossReferenceModel.curie == "FB:FBrf0221304").one()
+            xref = db.query(CrossReferenceModel).filter(CrossReferenceModel.curie == "FB:FBrf0221304").first()
```

Comment:
Wondering with previous comment, because before it became scalar() and here it's becoming first()

Reply:
The method .first() fetches the first row of the result set and returns it as an object since we are querying CrossReferenceModel instead of CrossReferenceModel.curie.

   we can also rewrite it to:

curie = db.query(CrossReferenceModel.curie).filter(CrossReferenceModel.curie == "FB:FBrf0221304").scalar()

Reply:
got it, thanks !